### PR TITLE
Enh/1/assignment1

### LINKED ISSUE #1: Module-1 Assignment-2
Exercises

Ensure you have SSH access to Tiger GitHub. This  allows you to work without having to type in your username/password constantly.
create a private repository named mle-training with an empty README.

Create a GitHub issue for the current assignment and add in the details based on what is requested in this assignment and assign it to yourself. Note the issue-id. 

Create a new branch (enh/<issue-id>/<short-msg>) and copy the [code-snippet here](https://drive.google.com/drive/folders/1ebqWUF06mkLRpmV-AjAjPMYgmh0cuUvX) and commit as-is including the README. Use appropriate commit messages as per the guidelines.

Add a .gitignore file to the repository to avoid committing unnecessary files like pyc files and the CSV data files in your current folder.  

Use Conda to set up a python development environment named mle-dev to run the script. Export the e
Environments file as env.yml and commit it to the repository.

Update the README file with instructions on how to run the code.

Create a Pull Request (PR). Make sure the PR follows the expected standards. [Link the PR to the GitHub issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to ensure the issue is closed automatically when the PR is merged.

Add the facilitators for your session as collaborators on the mle-training repository and request a PR.

Once the PR is approved, merge the PR to master branch and [give it a tag ](https://git-scm.com/book/en/v2/Git-Basics-Tagging)(named version) `v0.1` and push that to the remote.

### PR DESCRIPTION
**Pull request by Sreevathsava-TA**
This PR updates the README with instructions on how to run the Python script `nonstandardcode.py` to fetch and preprocess housing data. Closes #1 .

### CODE REVIEW

File: README.md
Comment (akashkumar-sharm) [inline README.md:1]: - add step/command on how to create an environment from your yml file
- and then try to create env again using your yml file and then run your python script and check if it works.

File: env.yml
Comment (akashkumar-sharm) [inline env.yml:1]: remove prefix

File: nonstandardcode.py
Comment (akashkumar-sharm) [inline nonstandardcode.py:1]: I am getting sinleton array error while running this script. Please resolve all issues in this script and push again.

**Review by akashkumar-sharm — CHANGES_REQUESTED** [review]

**Review by akashkumar-sharm — CHANGES_REQUESTED** [review]
Please look in to all the comments and work on the changes. then re-request for review.

**Review by akashkumar-sharm — CHANGES_REQUESTED** [review]
Add clear instruction in `Readme.md` file.
- add step/command on how to create an environment from your yml file
- and then try to create env again using your yml file and then run your python script and check if it works.

File: README.md
Comment (Sreevathsava-TA) [reply README.md:1]: Command to create an environment from .yml file 
conda env create -f env.yml
In the below file I have created env using my .yml file

Screenshot 2024-10-04 at 8 47 00 PM

Screenshot 2024-10-04 at 8 55 24 PM

**Review by akashkumar-sharm — APPROVED** [review]

**Pull request merged**